src/site/stages/build/drupal/metalsmith-drupal.js

```diff
@@ -24,16 +23,31 @@ const DRUPAL_COLORIZED_OUTPUT = chalk.rgb(73, 167, 222);
 // eslint-disable-next-line no-console
 const log = message => console.log(DRUPAL_COLORIZED_OUTPUT(message));
 
+function writeDrupalIndexPage(files) {
+  log('Drupal index page written to /drupal.');
+
+  const drupalPages = Object.keys(files)
+    .filter(page => page.startsWith('drupal'))
+    .map(page => `<li><a href="/${page}">/${page}</a></li>`)
+    .join('');
+
+  const drupalIndex = `
+    <h1>The following pages were provided by Drupal:</h1>
+    <ol>${drupalPages}</ol>
+  `;
+
+  files['drupal/index.html'] = {
+    contents: Buffer.from(drupalIndex),
+  };
+}
+
```

Comment:
This logic for creating the Drupal index page is just temporary code, so I isolated it into its own function.